lib/AdminPages/Pages/NewsletterEditor.php

```diff
@@ -106,4 +114,28 @@ public function render() {
 
     $this->pageRenderer->displayPage('newsletter/editor.html', $data);
   }
+
+  private function formatSubscriber(SubscriberEntity $subscriber): array {
+    return [
+      'id' => $subscriber->getId(),
+      'wp_user_id' => $subscriber->getWpUserId(),
+      'is_woocommerce_user' => (string)$subscriber->isWPUser(), // BC compatibility
```

Comment:
Why didn't you use the `getIsWoocommerceUser()` method? And why is this marked as BC compatibility?

Reply:
The usage of the wrong getter is a mistake. Thanks 👍🏻 
And the comment is as a note because the original type from the getter is boolean, and we want to type it as a string because it was used in the past.